sources/packages/backend/apps/api/src/route-controllers/institution/institution.controller.service.ts

```diff
@@ -86,4 +86,17 @@ export class InstitutionControllerService {
       description: institutionType.name,
     }));
   }
+
+  /**
+   * Get the list of all institutions names to be returned in an option
+   * list (key/value pair) schema.
+   * @returns institutions names in an option list (key/value pair) schema.
+   */
+  async getInstitutionNameOptions(): Promise<OptionItemAPIOutDTO[]> {
```

Comment:
In this scenario where the method would be unlikely to be shared between two clients, we can have small logic like this inside the controller itself.
The controller's services are intended to share logic between two clients or reduce the amount of code in the controller file.
This is not a blocker.